File: composeApp/src/commonMain/kotlin/net/matixmedia/og_meals/domain/model/Canteen.kt
package net.matixmedia.og_meals.domain.model


data class Canteen(
    val id: Long,
    val name: String,
    val hash: String,
    val displayName: String,
    val outletId: Int,
    val locationInfo: LocationInfo,
    val meals: List<Meal>
)

data class LocationInfo(
    val id: Long,
    val name: String
)

File: app/types/canteen.ts
export interface CanteenSummary {
  id: number
  name: string
  displayName: string
  orderInApp: number
}

export interface Canteen extends CanteenSummary {
  hash: string
  outletId: number
  locationInfo: { id: number; name: string }
}
